src/v/cloud_storage/remote_partition.cc

```diff
@@ -424,15 +456,13 @@ class partition_record_batch_reader_impl final
 
     ss::shared_ptr<remote_partition> _partition;
     ss::lw_shared_ptr<storage::offset_translator_state> _ot_state;
-    /// Currently accessed segment
-    remote_partition::iterator _it;
-    remote_partition::iterator _end;
     /// Reader state that was borrowed from the materialized_segment_state
     std::unique_ptr<remote_segment_batch_reader> _reader;
     /// Cancelation subscription
     ss::abort_source::subscription _as_sub;
     /// Guard for the partition gate
     gate_guard _gate_guard;
+    model::offset _next_offset{};
```

Comment:
A bit confusingly named. This is the base offset of the next segment, right?

Reply:
renamed it